src/components/views/rooms/MessageComposer.js

```diff
@@ -64,6 +65,13 @@ export default class MessageComposer extends React.Component {
         // XXX: fragile as all hell - fixme somehow, perhaps with a dedicated Room.encryption event or something.
         MatrixClientPeg.get().on("event", this.onEvent);
         this._roomStoreToken = RoomViewStore.addListener(this._onRoomViewStoreUpdate);
+
+        MatrixClientPeg.get().getMediaLimits().then((limits) => {
+            this._uploadLimits = limits;
+        }).catch(() => {
+            // HS can't or won't provide limits, so don't give any.
+            this._uploadLimits = {};
+        })
```

Comment:
This will re-fetch the limits every time the composer is mounted, ie. every time you change rooms, which probably isn't what you wanted.